webpack/components/PreupgradeReportsList/components/PreupgradeReportEntry.js

```diff
@@ -1,16 +1,64 @@
 import React from 'react';
-import { ListView } from 'patternfly-react';
+import { ListView, Grid, Icon } from 'patternfly-react';
 import PropTypes from 'prop-types';
 
-import { additionalInfo } from './helpers';
+import { presentSeverity, tagInfo, hasRemediations } from './helpers';
 
 const PreupgradeReportEntry = ({ entry }) => (
   <ListView.Item
     key={entry.id}
     className="listViewItem--listItemVariants"
     stacked
-    additionalInfo={additionalInfo(entry)}
-  />
+    description={entry.title}
+    additionalInfo={[
+      <ListView.InfoItem key="k-host">
+        <Icon type="pf" name="cluster" /> {entry.hostname}
+      </ListView.InfoItem>,
+      <ListView.InfoItem key="k-severity">
+        <Icon
+          type="pf"
+          name="resources-full"
+          style={{ color: presentSeverity(entry.severity) }}
+        />{' '}
```

Comment:
Is adding a whitespace nodes something patternfly recommends for creating a space between text and icon? Wouldn't it be better to use stylesheets for styling?